pkg/client/cli/intercept_state.go

```diff
@@ -189,7 +212,7 @@ func interceptMessage(r *connector.InterceptResult) string {
 	case connector.InterceptError_NO_PREVIEW_HOST:
 		msg = `Your cluster is not configured for Preview URLs.
 (Could not find a Host resource that enables Path-type Preview URLs.)
-Please specify one or more header matches using --match.`
+Please specify one or more header matches using --http-match.`
```

Comment:
Wow, I did a really bad job of searching for all of the `--match` references when I mad that change, didn't I?